src/api/entities/Asset/Permissions.ts

```diff
@@ -69,9 +67,9 @@ export class Permissions extends Namespace<Asset> {
   /**
    * Invite an Identity to be an agent with permissions over this Asset
    *
-   * @note this will create an AuthorizationRequest which has to be accepted by the target Account.
-   *   An Account or Identity can fetch its pending Authorization Requests by calling {@link Authorizations.getReceived}.
-   *   Also, an Account or Identity can directly fetch the details of an Authorization Request by calling {@link Authorizations.getOne}
+   * @note this will create an {@link AuthorizationRequest Authorization Request} which has to be accepted by the `target` Identity.
```

Comment:
```suggestion
   * @note this will create an {@link AuthorizationRequest | Authorization Request} which has to be accepted by the `target` Identity.
```